bluetooth/src/androidLibMain/kotlin/scanner/Scanner.kt

```diff
@@ -61,10 +62,10 @@ actual class Scanner internal constructor(
 ) : BaseScanner(permissions, connectionSettings, autoRequestPermission, autoEnableSensors, stateRepo) {
 
     class Builder(
+        private val applicationContext: Context = ApplicationHolder.applicationContext,
         private val bluetoothScanner: BluetoothLeScannerCompat = BluetoothLeScannerCompat.getScanner(),
-        private val bluetoothAdapter: BluetoothAdapter? = BluetoothAdapter.getDefaultAdapter(),
+        private val bluetoothAdapter: BluetoothAdapter? = (applicationContext.getSystemService(Context.BLUETOOTH_SERVICE) as BluetoothManager).adapter,
```

Comment:
Changed order for builder params to init bleAdapter from passed applicationContext (API breaking changed)

Reply:
Should be ok as this is only when people do not use defaults or named arguments.